Workflow automation/Get-SecureScoreData/README.MD

```diff
@@ -0,0 +1,20 @@
+# Get-SecureScoreData
+author: Nicholas DiCola
+
+This playbook uses the new Secure Score API to pull Secure Score, Controls and Assements into a Log Analytics workspace.
```

Comment:
Suggest to mention it will happens on a daily basis so the users will know what to expect.

Reply:
@mosheil done.